### ISSUE TITLE
Linking to FCC from the forum

### ISSUE THREAD

**Issue by m-henderson** [freeCodeCamp/discourse-meta#30]
We need to link to free code camp from the forum.

**Comment by raisedadead** [comment]
Yes sure we do.! How about adding a nav button? Certainly not the logo.

**Comment by m-henderson** [comment]
there is a link to the forum from the official site.

**Comment by raisedadead** [comment]
@m-henderson wasn't this issue for a NAV button the other way round, i.e going back to main website from forum.
 
> We need to link to free code camp from the forum.

**Comment by m-henderson** [comment]
darn, i didnt even think about that. I thought it was a button from the official site to the forum. We need to get a link on the forum to the site for sure!
